examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<ol><li>Please use "task" in place of "job", this is the Braket terminology.</li><li>"one should" -&gt; "we can" or "one can". We try to steer clear of sounding imperative. </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>